ui/src/journals/containers/SearchPageContainer.tsx

```diff
@@ -46,20 +46,25 @@ export const JournalSearchPage = ({
       <DocumentHead title={TITLE} description={META_DESCRIPTION} />
       <Row className="mt3" gutter={SEARCH_PAGE_GUTTER} justify="center">
         <Col xs={24} lg={16} xl={16} xxl={14}>
+          {/* @ts-ignore */}
```

Comment:
what exactly are we ignoring? Is that possible to add `@ts-ignore` for a specific thing?